p2p/dandelion-tower/src/pool/incoming_tx.rs

```diff
@@ -0,0 +1,110 @@
+//! Contains [`IncomingTx`] and [`IncomingTxBuilder`]
+use crate::{State, TxState};
+
+/// An incoming transaction that has gone through the preprocessing stage.
+pub struct IncomingTx<Tx, TxID, PID> {
```

Comment:
`TxId`, `Pid`?

Instead of enforcing this myself, can we add this to `deny`?: https://rust-lang.github.io/rust-clippy/master/index.html#/upper_case_acronyms

Reply:
Generic would fall under `Type parameters` so:

> concise `UpperCamelCase`, usually single uppercase letter: `T`

I'm guilty too...

https://github.com/Cuprate/cuprate/blob/59adf6dcf8a1adfbfbf05583866f6c5b0d85b374/storage/database/src/tables.rs#L275-L280

Reply:
Yeah sure we could test it